src/exes/moc.ml

```diff
@@ -194,14 +194,19 @@ let argspec = [
 
   (* persistence *)
   "--enhanced-orthogonal-persistence",
-  Arg.Unit (fun () -> Flags.enhanced_orthogonal_persistence := true),
-  " use enhanced orthogonal persistence (default): Scalable and fast upgrades using a persistent 64-bit main memory.";
+  Arg.Unit (fun () -> Flags.enhanced_orthogonal_persistence := true;
+                      Flags.explicit_enhanced_orthogonal_persistence := true),
+  " use enhanced orthogonal persistence (default): Scalable and fast upgrades using a persistent 64-bit main memory. Also, enable upgrade from classical to enhanced orthogonal persistence";
 
   (* persistence *)
   "--legacy-persistence",
   Arg.Unit (fun () -> Flags.enhanced_orthogonal_persistence := false),
   " use legacy (classical) persistence. This also enables the usage of --copying-gc, --compacting-gc, and --generational-gc. Deprecated in favor of the new enhanced orthogonal persistence, which is default. Legacy persistence will be removed in the future.";
 
+  "-unguarded-enhanced-orthogonal-persistence",
```

Comment:
why do we need that? a test without any flag is not enough?

Reply:
No, because all our tests are run with --legacy-persistence for now to avoid rewriting them all. This is a hack to restore implicit eop, just for the purpose of the test. Ugly, I know, but I would like to test the safeguard somehow... Any suggestions?

Reply:
Use an environment variable, like `//MOC-ENV MOC_UNLOCK_PRIM=yesplease`

Reply:
Tried, but turns ugly fast, so sticking to this solution